user_guide_src/source/libraries/encryption.rst

```diff
@@ -107,7 +106,14 @@ you can use the Encryption library's ``createKey()`` method.
 ::
 
 	// $key will be assigned a 32-byte (256-bit) random key
-	$key = Encryption::createKey(32);
+	$key = Encryption::createKey();
+
+	// $key will be assigned a 24-byte random key
+	$key = Encryption::createKey(24);
```

Comment:
Can we remove it for now? I don't want to encourage users to play with this variable if they don't know what they're doing.